docs/readthedocs/source/doc/LLM/Overview/KeyFeatures/gpu_supports.md

```diff
@@ -1,47 +1,170 @@
+         model = optimize_model(model) # With only one line to enable BigDL-LLM INT4 optimization
+
+         model = model.to('xpu') # Important after obtaining the optimized model
+
+      Especially, if you have saved the optimized model following setps `here <./optimize_model.html#save>`_, the loading process on Intel GPUs maybe as follows:
+
+      .. code-block:: python
+
+         import intel_extension_for_pytorch as ipex
+         from transformers import LlamaForCausalLM
+         from bigdl.llm.optimize import low_memory_init, load_low_bit
+
+         saved_dir='./llama-2-bigdl-llm-4-bit'
+         with low_memory_init(): # Fast and low cost by loading model on meta device
+            model = LlamaForCausalLM.from_pretrained(saved_dir,
+                                                     torch_dtype="auto",
+                                                     trust_remote_code=True)
+         model = load_low_bit(model, saved_dir) # Load the optimized model
+
+         model = model.to('xpu') # Important after obtaining the optimized model
+
+   .. tab:: ``transformers``-style API
+
+      You could run any Hugging Face Transformers model with ``transformers``-style API, and the loading and optimizing process on Intel GPUs maybe as follows:
+      
+      .. code-block:: python
+
+         # Take Llama-2-7b-chat-hf as an example
+         import intel_extension_for_pytorch as ipex
+         from bigdl.llm.transformers import AutoModelForCausalLM
+
+         # Load model in 4 bit, which convert the relevant layers in the model into INT4 format
+         model = AutoModelForCausalLM.from_pretrained('meta-llama/Llama-2-7b-chat-hf', load_in_4bit=True)
+
+         model = model.to('xpu') # Important after obtaining the optimized model
+
+      Especially, if you have saved the optimized model following setps `here <./hugging_face_format.html#save-load>`_, the loading process on Intel GPUs maybe as follows:
+
+      .. code-block:: python
+
+         import intel_extension_for_pytorch as ipex
+         from bigdl.llm.transformers import AutoModelForCausalLM
+
+         saved_dir='./llama-2-bigdl-llm-4-bit'
+         model = AutoModelForCausalLM.load_low_bit(saved_dir) # Load the optimized model
+
+         model = model.to('xpu') # Important after obtaining the optimized model
+```
+
+### Run Optimized Model
+
+You could then do inference using the optimized model on Intel GPUs almostly the same as on CPUs. **The only difference is to set `to('xpu')` for input tensors and `.cpu()` for output tensors.**
+
+Continuing with the [example of Llama-2-7b-chat-hf](#load-and-optimize-model), running as follows:
+```python
+import torch
+
+with torch.inference_mode():
+   prompt = 'Q: What is CPU?\nA:'
+   input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu') # With .to('xpu') specifically for inference on Intel GPUs
+   output = model.generate(input_ids, max_new_tokens=32)
+   output = output.cpu() # Move to CPU
```

Comment:
I recall that in some situations `output=output.cpu()` is not needed? anyway it does no harm to add this line, right?

Reply:
Have separated into two pages. 

> I recall that in some situations `output=output.cpu()` is not needed? anyway it does no harm to add this line, right?

This line may cause errors sometimes (for example, if we run bark model through TTS, the output maybe `numpy.ndarray`, then `.cpu()` may cause error). We only use it to move tensors to CPU, and it is not a necessary operation, so I remove it now.